karma.debug.conf.js

```diff
@@ -0,0 +1,16 @@
+// Karma configuration
+
+module.exports = function (config) {
+    'use strict';
+
+    var commonConfig = require('./karma.common.conf');
+
+    commonConfig(config, {
+        singleRun: false,
+        autoWatch: true,
+        browsers: ['Chrome'],
```

Comment:
Is there a way that we can run automated tests with Firefox? Maybe we could create a karma.ci.conf.js which runs all browsers, not just WebKit-based ones in the CI environment.